src/gui/MainWindow.cpp

```diff
@@ -822,18 +826,24 @@ void MainWindow::onActiveDocumentChanged(const int newIndex){
             for(QAction * action:singleViewAct) action->setChecked(false);
             singleViewAct[documents[activeDocumentId]->getDisplayGrid()->getActiveDisplayId()]->setChecked(true);
         }
-
+    }
+    else if (displayGrid == nullptr && documentArea->count() > 1) {
```

Comment:
Why isn't displayGrid == nullptr enough here?

Reply:
It just crashes the program and the program is unable to start. During debugging, I found that whenever the program starts,
<HTML>
<head>

<body>
<!--StartFragment-->


  | Name | Value | Type
-- | -- | -- | --
▶ | displayGrid | 0x0000000000000000 <NULL> | DisplayGrid *


<!--EndFragment-->
</body>
</HTML>


Then it calls `Dockable::clear()` where it checks for `widget()` and an exception is throwed. Here is the exception:
```Exception thrown at 0x00007FFB9E9157AA (Qt5Widgetsd.dll) in arbalest.exe: 0xC0000005: Access violation reading location 0xFFFFFFFFFFFFFFFF.```
This concludes we are trying to read a variable that is `NULL` or set to `NULL`.
```cpp
void Dockable::clear() {
    if(widget()){
        widget()->setVisible(false);
        widget()->setParent(nullptr);
    }
    setContent(&fillerWidget);
}
```